kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/drd/DRDNameChangerView.java

```diff
@@ -137,7 +135,16 @@ void saveForTheCurrentDiagram() {
     }
 
     private void performSave(final DMNDiagramElement dmnDiagramElement) {
-        dmnDiagramElement.getName().setValue(drdNameInput.getValue());
-        selectedEvent.fire(new DMNDiagramSelected(dmnDiagramElement));
+        if (!dmnDiagramElement.getName().getValue().equals(drdNameInput.getValue())) {
```

Comment:
It is a minor thing, but I prefer `Objects.equals(a, b)` rather than `a.equals(b)`. as the first mentioned shouldn't throw a `NullPointerException` in case `a` or `b` is null`.
```
if (!Objects.equals(dmnDiagramElement.getName().getValue(), drdNameInput.getValue())) {
```

Reply:
@jomarko Done!